apps/aws-amplify/src/locations/ConfigScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import { useCallback, useState, useEffect, ChangeEvent } from 'react';
+import { ConfigAppSDK } from '@contentful/app-sdk';
+import {
+  Box,
+  Flex,
+  FormControl,
+  Heading,
+  HelpText,
+  Paragraph,
+  Stack,
+  TextInput,
+  TextLink,
+} from '@contentful/f36-components';
+import { css } from 'emotion';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import tokens from '@contentful/f36-tokens';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import AmplifyIcon, { AWSAmplifyBrand } from '../components/common/AmplifyIcon';
+import { isValidUrl } from '../lib/isVaildUrl';
+
+export interface AppInstallationParameters {
+  amplifyWebhookUrl?: string;
+}
+
+export const styles = {
+  body: css({
+    height: 'auto',
+    minHeight: '40vh',
+    margin: '0 auto',
+    marginTop: tokens.spacingXl,
+    padding: `${tokens.spacingXl} ${tokens.spacing2Xl}`,
+    maxWidth: '900px',
+    backgroundColor: tokens.colorWhite,
+    zIndex: 2,
+    boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.1)',
+    borderRadius: '2px',
+  }),
+  background: css({
+    display: 'block',
+    position: 'absolute',
+    zIndex: -1,
+    top: 0,
+    width: '100%',
+    height: '300px',
+    backgroundColor: AWSAmplifyBrand.primaryColor,
+  }),
+  box: {
+    width: '100%',
+    marginTop: '22px',
+  },
+  icon: {
+    marginTop: '41px',
+  },
+};
+
+const ConfigScreen = () => {
+  const sdk = useSDK<ConfigAppSDK>();
+  const [parameters, setParameters] = useState<AppInstallationParameters>({});
+  const [amplifyWebhookUrl, setAmplifyWebhookUrl] = useState<string>('');
+
+  const onConfigure = useCallback(async () => {
+    const currentState = await sdk.app.getCurrentState();
+
+    if (!isValidUrl(amplifyWebhookUrl)) {
+      sdk.notifier.error('Please provide a valid webhook URL.');
+      return false;
+    }
+
+    const parametersToSave = {
+      ...parameters,
+      amplifyWebhookUrl,
+    };
+
+    setParameters(parametersToSave);
+
+    return {
+      parameters: parametersToSave,
+      targetState: currentState,
+    };
+  }, [parameters, sdk, amplifyWebhookUrl]);
+
+  useEffect(() => {
+    sdk.app.onConfigure(() => onConfigure());
+  }, [sdk, onConfigure]);
+
+  useEffect(() => {
+    (async () => {
+      const currentParameters: AppInstallationParameters | null = await sdk.app.getParameters();
```

Comment:
Pretty sure you can just not type this at all (basically remove `AppInstallationParameters | null` and this will all work just fine

Reply:
this came from the boilerplate and while it implicitly assumes the correct types because of the return value, I don't think it hurts to have the visible definition for readability. I can definitely remove it if we feel strongly about it though!

Reply:
I went a head and removed it actually 👍 will rely on the implict types